test/suites/fips/validate-fips.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation       Tests related to FIPs Validation
+
+Resource            ../../resources/ostree-health.resource
+Resource            ../../resources/common.resource
+Resource            ../../resources/selinux.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${USHIFT_HOST}      ${EMPTY}
+${USHIFT_USER}      ${EMPTY}
+
+
+*** Test Cases ***
+FIPs Validation
+    [Documentation]    Performs a FIPs validation against the host
+    Wait Until Greenboot Health Check Exited
+    Fips Should Be Enabled
+
+Microshift Fips Binary
+    [Documentation]    Performs a FIPs validation against the microshfit binary
+    Microshift Binary Should Dynamicly Link OpenSSL
+
+
+*** Keywords ***
+Setup
+    [Documentation]    Test suite setup
+    Check Required Env Variables
+    Login MicroShift Host
+
+Teardown
+    [Documentation]    Test suite teardown
+    Logout MicroShift Host
+
+Microshift Binary Should Dynamicly Link OpenSSL
```

Comment:
Nit: Is the name of the function precise? We're checking the FIPS library link here, aren't we?

Reply:
to be precise FIPs is openssl module, fixed